Clinical-Genomics/scout
# Support for custom public igv tracks

This PR is one of a series of 2 (the second PR will add support for private tracks)
It adds the support for including any public track (immediately downloadable link) into the list of available tracks that can be shown in the igv.js browser.

**How to test**:
1. Modify the scout config file by adding these lines (I'm hosting a test track in my S3 storage, just for testing the PR)
```
CLOUD_IGV_TRACKS = [
    {
        "name": "custom_public_bucket",
        "access": "public",
        "tracks": [
            {
                "name": "dbVar Pathogenic or Likely Pathogenic",
                "type": "variant",
                "format": "vcf",
                "build": "37",
                "url": "http://public-igv-tracks-scout.s3.eu-north-1.amazonaws.com/GRCh37.variant_call.clinical.pathogenic_or_likely_pathogenic.vcf.gz",
                "indexURL": "http://public-igv-tracks-scout.s3.eu-north-1.amazonaws.com/GRCh37.variant_call.clinical.pathogenic_or_likely_pathogenic.vcf.gz.tbi",
            }
        ],
    },
]
```
1. Install the branch and make sure that the `dbVar Pathogenic or Likely Pathogenic` track appears among the customizable tracks of a variant:
![image](https://user-images.githubusercontent.com/28093618/95331079-c1de5200-08a9-11eb-969c-6689509a519a.png)

1. Make sure that is the track is selected, then it's shown in igv.js:
![image](https://user-images.githubusercontent.com/28093618/95331212-f3571d80-08a9-11eb-9329-26b30b94f10e.png)


**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
